docs/content/querying/select-query.md

```diff
@@ -179,3 +179,14 @@ Note that in the second query, an offset is specified and that it is 1 greater t
     "pagingSpec":{"pagingIdentifiers": {}, "threshold":5, "fromNext": true}
   }
 ```
+
+Note that specifying the `fromNext` option in the `pagingSpec` overrides the default value set by `druid.query.select.enableFromNextDefault` in the server configuration. See [Server configuration](#server-configuration) for more details.
```

Comment:
I rewrote part of the docs to better explain the query pagination and fromNext, with mention of backwards compatibility